pkg/volume/glusterfs/glusterfs.go

```diff
@@ -438,6 +577,24 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 
 	glog.V(4).Infof("glusterfs: deleting volume %q with configuration %+v", volumeId, d.provisioningConfig)
 
+	gid, err := d.getGid()
+	if err != nil && err != ErrNotFound {
+		glog.Errorf("glusterfs: failed to parse the gid")
+		return fmt.Errorf("glusterfs: failed to parse the gid")
```

Comment:
what does recovery look like in this situation? they manually fix up the annotation on the PV and we try to delete it later? are we ok blocking deletion until that happens? wondering if we should just log or call util.HandleError() and continue deleting